.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  "env": {
+      "es6": true,
+      "node": true
+  },
+  "ignorePatterns": [
```

Comment:
why do we need both this and .eslintignore?

Reply:
Good point. We don't. Fixed in ffcd79b